doc/src/devdocs/inference.md

```diff
@@ -13,8 +13,8 @@ posts
 
 You can start a Julia session, edit `compiler/*.jl` (for example to
 insert `print` statements), and then replace `Core.Compiler` in your
-running session by navigating to `base/compiler` and executing
-`include("compiler.jl")`. This trick typically leads to much faster
+running session by navigating to `base` and executing
+`include("compiler/compiler.jl")`. This trick typically leads to much faster
 development than if you rebuild Julia for each change.
```

Comment:
Should also mention `Revise` and `Revise.track(Core.Compiler)`

Reply:
That's a great idea, is the updated version OK?